DEVELOP.md

```diff
@@ -76,6 +96,24 @@ Where `SIGNING_KEY_ID` is the key id associated with the signing key,
 `OSSRH_USERNAME` is the Sonatype user name to upload the repository to,
 and `OSSRH_PASSWORD` is the password for the Sonatype account to upload the repository to.
 
+If these values are present as environment variables they will also be picked up.
+
+#### Publishing via CI
+
+BitRise automatically publishes every branch to the Sonatype snapshot repository (`https://oss.sonatype.org/content/repositories/snapshots/`). This is done using a (private) BitRise project. A Maven Central release can be done by manually starting the `publisMavenCentralRelease` workflow for the appropriate release branch.
+
+#### Releasing to Maven Central
+
+Projects publishing to Sonatype's release repository need to be manually closed and released before they will appear on Maven Central. This can only be done by people with access.
+
+#### Increase version after publishing
+
+in case this has not been done yet, bump the version at [gradle/ext.gradle](gradle/ext.gradle) in the `develop` branch to start the next development iteration.
```

Comment:
```suggestion
In case this has not been done yet, bump the version at [gradle/ext.gradle](gradle/ext.gradle) in the `develop` branch to start the next development iteration.
```